style.css

```diff
@@ -0,0 +1,98 @@
+.body {
+  overflow-x: hidden;
+}
+
+.nav_container {
+  display: flex;
+  width: 100%;
+  margin-left: -25px;
+  margin-top: -8px;
+}
+
+.nav_container,
+.logo,
+.toggle {
+  display: flex;
+  padding: 20px;
+}
+
+.logo {
+  width: 25%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.logo img {
+  width: 100%;
+}
+
+.toggle img {
+  width: 50%;
+}
+
+.toggle {
+  width: 10%;
+  margin-left: auto;
+  align-items: center;
+  justify-content: center;
+}
+
+.main_container {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  background-image: url("images/header-shapes-mobile.png");
+  background-repeat: no-repeat;
+  background-size: 100%;
+}
+
+.box1 {
+  width: 80%;
+  margin-top: 20%;
+  font-family: Georgia, 'Times New Roman', Times, serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 80px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: #172b4d;
+  padding: 24px;
+}
+
+.box2 {
+  font-family: Georgia, 'Times New Roman', Times, serif;
+  font-style: normal;
+  font-size: 25px;
+  vertical-align: top;
+  color: #344563;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 24px;
+}
+
+.box3 {
+  font-family: Georgia, 'Times New Roman', Times, serif;
+  font-style: normal;
+  line-height: 24px;
+  font-size: 27px;
+  vertical-align: top;
+  color: #7f8cff;
+  display: flex;
+  align-items: flex-start;
+  padding: 24px;
+}
+
+.box4 {
+  display: flex;
+  padding: 24px;
+}
+
+.socials {
+  width: 10%;
+  align-items: center;
+  justify-content: center;
+  padding: 10px;
+}
```

Comment:
- [ ] Please consider adjusting some of the styles properties you have here to match with your template on Figma. I suggest you take a look at the design template you chose on Figma and follow the style properties there. like `margins`

Your design:
<img width="234" alt="Loom_lQCteSfGDd" src="https://user-images.githubusercontent.com/81468675/148227635-8835d65a-b5ac-4698-ad25-9a8865bc1d1b.png">

Figma design:
<img width="238" alt="Loom_0Q5IfoCinJ" src="https://user-images.githubusercontent.com/81468675/148227684-1bea2ed4-9c99-408d-884e-09d1bad209c2.png">